spec/services/participants/resume_spec.rb

```diff
@@ -35,7 +35,7 @@
     end
 
     context "when a matching application does not exist (different lead provider)" do
-      let(:lead_provider) { create(:lead_provider) }
+      let(:lead_provider) { LeadProvider.all.excluding(application.lead_provider).sample }
```

Comment:
I think @ethax-ross has fixed this in his pr :P https://github.com/DFE-Digital/npq-registration/pull/1430/files#diff-27987830d045f99f9132c178a2e7640f50db10a8e4f50e580e8b53fd8fc00c0bR28